support-frontend/app/views/subscriptionRedemptionForm.scala.html

```diff
@@ -28,10 +27,9 @@
     <script type="text/javascript">
       window.guardian = window.guardian || {};
       window.guardian.stage = "@stage";
-      window.guardian.userCode = "@redemptionCode";
-      window.guardian.error = "@customerOrError.left.getOrElse("")";
+      window.guardian.userCode = "@Html(redemptionCode.replaceAll(""""""", """\\""""))";
+      window.guardian.error = "@Html(maybeRedemptionError.getOrElse("").replaceAll(""""""", """\\""""))";
```

Comment:
this should escape the strings properly into the JS without doing html escaping (i.e. ending up with &quot; and &amp; etc.

Reply:
Yes I would extract this to a helper function